README.adoc

```diff
@@ -17,63 +17,22 @@ image:https://travis-ci.com/kiegroup/optaweb-vehicle-routing.svg?branch=master[
 Web application for solving the https://www.optaplanner.org/learn/useCases/vehicleRoutingProblem.html[Vehicle Routing Problem]
 using https://www.optaplanner.org/[OptaPlanner].
 
-== Running the Application
+== Starting the Application
 
 First, clone this repository and `cd` into it.
 
-Frontend and backend are built and run independently. Open two terminal windows.
+Frontend and backend are built and run independently. Open two terminal windows. Start the backend first, and then, when the backend is ready, start the frontend.
 
-== Run Frontend
+=== Run Backend
```

Comment:
Strictly speaking, ,"frontend" and "backend" are off limits in our style, but again, I see in general that the content in this file is all pretty loose and casual, and community-voiced, so I guess whatever.